src/components/Observations/map/ObservationMapView.tsx

```diff
@@ -32,7 +30,13 @@ export default function ObservationMapView({ observationsResults, search }: Obse
   const [selectedObservationDate, setSelectedObservationDate] = useState<string | undefined>();
   useEffect(() => {
     if (observationsDates) {
-      setSelectedObservationDate(observationsDates[observationsDates.length - 1]);
+      setSelectedObservationDate((currentDate) => {
+        if (!currentDate || !observationsDates.includes(currentDate)) {
+          return observationsDates[observationsDates.length - 1];
```

Comment:
can observationDates be empty?

Reply:
I think it can, and I should make sure I account for that. Thanks